docs/source/tools/extractor.rst

```diff
@@ -59,7 +59,7 @@ Running the Extractor
 
 The basic command to run the Extractor is::
 
-  $ da run extractor -- [options]
+  $ daml extractor -- [options]
```

Comment:
I don’t think we include `extractor` in the release tarball atm. Should be easy to fix.

Reply:
Just checked, I was wrong. We do include `extractor` and I was even the one that added it :man_facepalming: